www/src/pages/foundations/typography.tsx

```diff
@@ -20,6 +20,65 @@ const weightLabels: WeightLabelsTypes = {
   800: 'Black',
 };
 
+const headingSizes = [1, 2, 3, 4, 5, 6];
+
+const displaySizes = [1, 2, 3, 4];
+
+const decorationAndEmphasisClassesAndDescriptions = [
+  { className: 'text-lowercase', text: 'Lowercase text.' },
+  { className: 'text-uppercase', text: 'uppercase text.' },
+  { className: 'text-capitalize', text: 'capitalize text.' },
+  { className: 'text-decoration-none', text: 'No decorations.' },
+  { className: 'font-italic', text: 'Italic text.' },
+  { className: 'font-weight-bold', text: 'Bold text.' },
+  { className: 'font-weight-normal', text: 'Regular text.' },
+];
+
+const alignmentClassesAndDescriptions = [
+  { className: 'text-left', text: 'left text.' },
+  { className: 'text-right', text: 'right text.' },
+  { className: 'text-center', text: 'center text.' },
+  {
+    className: 'text-justify',
+    text: 'The text-justify class specifies the justification method of text. This text-justify class spreads the words into the complete line with equal spaces.',
+  },
+  {
+    className: 'text-wrap',
+    text: 'Use text-wrap to cause text to wrap normally within an element. Newlines and spaces will be collapsed.',
+  },
+  {
+    className: 'text-nowrap',
+    text: 'Use text-nowrap to prevent text from wrapping within an element. Newlines and spaces will be collapsed. You can prevent line breaks and text wrapping for specific elements'
+  },
+];
+
+const bodyClassesAndDescriptions = [
+  { className: 'lead', text: 'Large Body' },
+  { className: '', text: 'Body' },
+  { className: 'small', text: 'Small Body' },
+  { className: 'x-small', text: 'Extra Small Body' },
+  { className: 'micro', text: 'Micro Body' },
+];
+
+const linksClassesAndDescriptions = [
+  {
+    name: 'Standalone Link',
```

Comment:
you can wrap both `name` and `text` into `span` tag or something else and just render it instead of using `_dangerouslySetInnerHTML`

Reply:
Thanks